--- tesseract-ocr/dict/permdawg.cpp~	2012-09-02 22:08:43.000000000 +0200
+++ tesseract-ocr/dict/permdawg.cpp	2013-07-14 16:06:15.545850330 +0200
@@ -205,7 +205,7 @@
           STRING word_str;
           word->string_and_lengths(&word_str, NULL);
           word_str += " ";
-          fprintf(output_ambig_words_file_, word_str.string());
+          fprintf(output_ambig_words_file_, "%s", word_str.string());
         }
         WERD_CHOICE *adjusted_word = word;
         WERD_CHOICE hyphen_tail_word(&getUnicharset());
