--- tesseract-3.04.01/opencl/Makefile.am.orig	2016-02-16 22:27:01.000000000 +0100
+++ tesseract-3.04.01/opencl/Makefile.am	2016-02-26 15:40:52.609300973 +0100
@@ -4,12 +4,14 @@
 
 if !USING_MULTIPLELIBS
 noinst_LTLIBRARIES = libtesseract_opencl.la
+libtesseract_opencl_la_LIBADD = $(OPENCL_LDFLAGS) -ltiff
 else
 lib_LTLIBRARIES = libtesseract_opencl.la
 libtesseract_opencl_la_LDFLAGS = -version-info $(GENERIC_LIBRARY_VERSION) $(OPENCL_LDFLAGS)
 libtesseract_opencl_la_LIBADD = \
     ../ccutil/libtesseract_ccutil.la \
-    ../viewer/libtesseract_viewer.la
+    ../viewer/libtesseract_viewer.la \
+    $(OPENCL_LDFLAGS) -ltiff
 endif
 
 libtesseract_opencl_la_SOURCES = \
